app/src/main/res/layout/fragment_movie_details.xml

```diff
@@ -81,69 +81,69 @@
             android:textAppearance="@style/movie_genre"
             android:textSize="@dimen/size_of_headers"
             app:layout_constraintStart_toStartOf="parent"
-            app:layout_constraintTop_toBottomOf="@+id/textViewNameMove" />
+            app:layout_constraintTop_toBottomOf="@+id/textViewDetailFragmentNameMove" />
 
 
         <ImageView
-            android:id="@+id/imageViewStarOneIcon"
-            android:layout_width="wrap_content"
-            android:layout_height="wrap_content"
+            android:id="@+id/imageViewDetailFragmentStarOneIcon"
+            android:layout_width="13dp"
+            android:layout_height="12dp"
             android:layout_gravity="center"
             android:layout_marginStart="@dimen/margin_star_of_parent"
-            android:layout_marginTop="4dp"
+            android:layout_marginTop="8dp"
```

Comment:
Why not use dimens.xml here (in this file other occurrences also)?

Reply:
fixed